app/assets/stylesheets/components/_form.scss

```diff
@@ -72,9 +72,10 @@ textarea {
 
 // hide number field input spin box as per:
 // http://stackoverflow.com/questions/3790935/can-i-hide-the-html5-number-input-s-spin-box
-// and added .mfa class selector as per CodeClimate warning to:
+// and added .mfa-alpha/mfa-digits class selectors as per CodeClimate warning to:
 // 'Avoid qualifying attribute selectors with an element.'
-.mfa {
+.mfa-alpha,
+.mfa-digits {
   -moz-appearance: textfield;
```

Comment:
To previous comment, I think this only mattered for if we rendered these as `type="number"`? Feels like we could be safe to remove it.